test/whl-scale-tests/prepare-geneva-xml.ps1

```diff
@@ -0,0 +1,46 @@
+param(
+    [string] [Parameter(Mandatory = $true)] $GenevaAccountName,
+    [string] [Parameter(Mandatory = $true)] $GenevaLogAccountNamespace,
+    [string] [Parameter(Mandatory = $true)] $GenevaMetricAccountName,
+    [guid] [Parameter(Mandatory = $true)] $AKSClusterMSIObjectId
+)
+
+$orignalPath = Get-Location
+Set-Location -Path $PSScriptRoot
+. .\common.ps1
+
+$genevaXmlConfigurationHashTable = @{
+    'GENEVA_ACCOUNT_NAME' = $GenevaAccountName;
+    'GENEVA_LOG_ACCOUNT_NAMESPACE' = $GenevaLogAccountNamespace;
+    'GENEVA_METRIC_ACCOUNT' = $GenevaMetricAccountName;
+    'CLUSTER_MANAGED_IDENTITY_OBJECT_ID' = $AKSClusterMSIObjectId;
+    'GENEVA_STORAGE_DIAG' = $GenevaLogAccountNamespace.ToLower()+"diag";
+    'GENEVA_STORAGE_SECURITY' = $GenevaLogAccountNamespace.ToLower()+"security";
+    'GENEVA_STORAGE_LOGSAUDIT' = $GenevaLogAccountNamespace.ToLower()+"logsaudit";
+}
+
+#loop through all files under geneva-config-files then update each
+$folderPath = ".\geneva-config-files" 

```

Comment:
Does the current geneva config collect the ETW Logs being generated by this container?

Reply:
Yes it does